cmd/podman/root.go

```diff
@@ -191,8 +191,9 @@ func loggingHook() {
 
 func rootFlags(opts *entities.PodmanConfig, flags *pflag.FlagSet) {
 	// V2 flags
-	flags.StringVarP(&opts.Uri, "remote", "r", registry.DefaultAPIAddress(), "URL to access Podman service")
-	flags.StringSliceVar(&opts.Identities, "identity", []string{}, "path to SSH identity file")
+	flags.StringVarP(&opts.Uri, "remote", "r", registry.DefaultAPIAddress(), "URL to access Podman servic (PODMAN_HOST)")
```

Comment:
I would prefer this be handled as two different options.  Since we can now store or soon will be able store the remoteuri in containers.conf we need a way to change podman to use the remote protocol
podman --use-remote
podman --client

Or do we change remote to be a bool and add --remote-uri